tools/run_tests/run_tests.py

```diff
@@ -884,6 +884,38 @@ def runs_per_test_type(arg_str):
         msg = '\'{}\' is not a positive integer or \'inf\''.format(arg_str)
         raise argparse.ArgumentTypeError(msg)
 
+_COMPILER_CHOICES = ['default',
```

Comment:
Never write a list literal when a tuple literal will do, especially for a constant.